### PR TITLE
feat: add new function for uri_map_to_string

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: src/emqx_http_lib.app.src [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {application, emqx_http_lib,
  [{description, "EMQX HTTP Library"},
-  {vsn, "0.4.1"},
+  {vsn, "0.4.2"},
```
Comment (zmstone) [inline src/emqx_http_lib.app.src]: 0.5.0

File: src/emqx_http_lib.erl [hunk 2]
```diff
@@ -19,14 +19,15 @@
 -export([ uri_encode/1
         , uri_decode/1
         , uri_parse/1
+        , uri_map_to_string/1
```
Comment (zmstone) [inline src/emqx_http_lib.erl]: normalize

File: src/emqx_http_lib.app.src (hunk 1 above)
Comment (Rory-Z) [reply src/emqx_http_lib.app.src]: Done

File: src/emqx_http_lib.erl (hunk 2 above)
Comment (Rory-Z) [reply src/emqx_http_lib.erl]: Done

**Pull request merged**